db/index.js

```diff
@@ -1,7 +1,7 @@
 require('dotenv').config();
 
 const Sequelize = require('sequelize');
-const sequelize = new Sequelize(process.env.MYSQL_DATABASE, process.env.MYSQL_USER, process.env.MYSQL_PASSWORD, {
+const sequelize = new Sequelize(process.env.RDS_HOSTNAME, process.env.RDS_USERNAME, process.env.RDS_PASSWORD, {
```

Comment:
Why are these changing? I don't see these environment variables in any `.env.local` or config files

Reply:
These are the database env variables that EB auto-exposes to our app using AWS RDS.

Reply:
This will make development db unusable. Can you add a flag based on if this is prod. Testing or dev?